src/app/PDF.tsx

```diff
@@ -196,6 +197,8 @@ const useHandlers = (): Handlers => {
       return fieldLayerHandlers(state, dispatch, createAnnotationAttr);
     case 1:
       return labelLayerHandlers(state, dispatch, createAnnotationAttr);
+    case 2:
```

Comment:
What do you think about changing each of these functions into hooks and then calling them inside the three components where we use them respectively?